src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -34,15 +37,26 @@ type Props = {
 const DefaultTranscriptslist = (props: Props) => {
   const { gene } = props;
   const sortedTranscripts = defaultSort(gene.transcripts);
+  const [isFilterOpen, setFilterOpen] = useState(false);
+
+  const toggleFilter = () => { isFilterOpen ? setFilterOpen(false) : setFilterOpen(true); }
 
   return (
     <div>
       <div className={styles.header}>
+        {isFilterOpen && <GeneFilter toggleFilter={toggleFilter} />}
         <div className={styles.row}>
-          <div className={styles.left}>Filter & sort</div>
-          <div className={styles.middle}></div>
+          <div className={isFilterOpen ?  styles.hidden : styles.filterLabel}>
+            Filter & sort
+            <button onClick={toggleFilter} className={styles.expandBtn}>
+              <img
+                src={chevronDownIcon}
+                alt={'collapse'}
+              />
```

Comment:
We usually use svg elements, because then we can style their `fill` property via CSS. If you run:

```
import {ReactComponent as ChevronDown} from 'static/img/shared/chevron-down.svg';
```
you'll get a react component called `ChevronDown`, which will be an `svg` element.